content/java/concepts/array-list/terms/trimToSize/trimToSize.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.trimToSize()'
+Description: 'Adjusts the capacity of the ArrayList to be the same as its size.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Data Types'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.trimToSize()`** method of the `ArrayList` class adjusts the capacity of the list to be the same as its size.
```

Comment:
Hyperlink the `ArrayList` with [this](https://www.codecademy.com/resources/docs/java/array-list) docs url.